### ISSUE TITLE
 Is there any reason not to merge _Storage and _Storage_root?

### ISSUE THREAD

**Issue by nritsche** [radiocosmology/caput#116]
@kiyo-masui after you had good arguments against the last proposal of changing the design, I'm hoping to get your opinion once again (again).

According to @jrs65 there's no good reason anymore to keep `_Storage` and `_StorageRoot` two separate classes.

**Comment by kiyo-masui** [comment]
I would merge this issue into #115. No point in tackling them separately and once you figure out how these classes work you'll know the answer to the question.

**Comment by jrs65** [comment]
Yeah. I agree with Kiyo here. I think the whole module needs some refactoring to reduce the structural complexity. These items were two of the things I thought possible, but they should really be considered in combination.

**Comment by nritsche** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [caput refactoring](https://trello.com/c/I2LkWOIe/6-caput-refactoring)

**Comment by jrs65** [comment]
@nritsche can you just use a Github project board rather than trello? It's easier to keep track of everything that way.
